packages/teams-js/src/public/barCode.ts

```diff
@@ -0,0 +1,82 @@
+import { sendAndHandleSdkError } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { validateScanBarCodeInput } from '../internal/mediaUtil';
+import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
+import { DevicePermission, ErrorCode } from './interfaces';
+import { runtime } from './runtime';
+
+/**
+ * Namespace to interact with the scan barCode module-specific part of the SDK.
+ */
+export namespace barCode {
+  /**
+   * Barcode configuration supplied to scanBarCode API to customize barcode scanning experience in mobile
+   * All properties in BarCodeConfig are optional and have default values in the platform
+   */
+  export interface BarCodeConfig {
+    /**
+     * Optional; Lets the developer specify the scan timeout interval in seconds
+     * Default value is 30 seconds and max allowed value is 60 seconds
```

Comment:
Suggestion:

/**
* Optional; designates the scan timeout interval in seconds.
*  Default value is 30 seconds, max allowed value is 60 seconds.